### PR TITLE
Fix Heartbreaker Toxin and Lexorin descriptions

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

I forgot to update the chemical descriptions for lexorin and heartbreaker toxin when i made the chemical synthesis PR.

### CODE REVIEW

File: Resources/Locale/en-US/reagents/meta/toxins.ftl [hunk 1]
```diff
@@ -26,10 +26,10 @@ reagent-name-unstable-mutagen = unstable mutagen
 reagent-desc-unstable-mutagen = Causes mutations when injected into living people or plants. High doses may be lethal, especially in humans.
 
 reagent-name-heartbreaker-toxin = heartbreaker toxin
-reagent-desc-heartbreaker-toxin = A hallucinogenic compound derived from mindbreaker toxin. it blocks neurological signals to the respiratory system, causing asphyxiation.
+reagent-desc-heartbreaker-toxin = A hallucinogenic compound derived from mindbreaker toxin. it blocks neurological signals to the respiratory system, causing asphyxiation. Dexalin, dexalin plus and epinephrine will filter it out, however.
```
Comment (Partmedia) [inline Resources/Locale/en-US/reagents/meta/toxins.ftl:29]: Could you please double check that this is also true for dexalin? I can confirm that dexalin plus and epi do this "filtering out".

Comment (UbaserB) [reply Resources/Locale/en-US/reagents/meta/toxins.ftl:29]: If dexalin does this, then it is not supposed to. Afterall, i was the one that removed their effect for lexorin and changed it over to be heartbreaker toxin. Will double check anyway.

Comment (Partmedia) [reply Resources/Locale/en-US/reagents/meta/toxins.ftl:29]: To be clear, you write "Dexalin... will filter it out..."

But it does not. So I'm asking you to double check that it doesn't and if not, update the description. Thanks :)

**Comment by UbaserB** [comment]
I double checked and it was true that normal dexalin doesn't purge. I have fixed this.

**Review by Partmedia — APPROVED** [review]

**Comment by Partmedia** [comment]
Thank you for the fixes.

**Pull request merged**